opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp

```diff
@@ -1947,9 +1950,9 @@ namespace {
             RG_sum = RG.sum();
         }
 
-        const double mass_balance_residual_water = fabs(BW_avg*RW_sum) * dt / pvSum;
-        const double mass_balance_residual_oil = fabs(BO_avg*RO_sum) * dt / pvSum;
-        const double mass_balance_residual_gas = fabs(BG_avg*RG_sum) * dt / pvSum;
+        const double mass_balance_residual_water = std::abs(BW_avg*RW_sum) * dt / pvSum;
+        const double mass_balance_residual_oil   = std::abs(BO_avg*RO_sum) * dt / pvSum;
+        const double mass_balance_residual_gas   = std::abs(BG_avg*RG_sum) * dt / pvSum;
```

Comment:
I will try to use explicit std::abs from now on. I guess my reluctance stems from a time when it was common to put `using namespace std` everywhere. I still think this is acceptable for cpp files (but have not done it in a decade or so I think, and will not recommend it), but it used to be commonplace enough to make fabs() safer than abs(), since most used unqualified function names.